### ISSUE TITLE
crates and lanterns gone?

### ISSUE THREAD

**Issue by leosprouts** [svenhjol/Charm#814]
hi, my buddies and i recently updated our server to 1.19 and to our dismay crates, a lot of the lanterns we used, copper rails, and other things we used in a lot of our builds were gone! was this intentional or did we mess something up?

**Comment by gander-code** [comment]
Crates seem to be missing for me as well, no mention of a removal on the curseforge page. I'm using 1.19.2. Not a huge deal but some communication about it would be appreciated.

**Comment by leosprouts** [comment]
yeah we're using the fabric version too! it's no biggie for us either, we found other options for storage and lighting and such, but some kind of communication/documentation would be great
